README.md

```diff
@@ -10,62 +10,69 @@
+  - [Deploy a released build](#deploy-a-released-build)
+  - [Build from source](#build-from-source)
+    - [Undeploy controller](#undeploy-controller)
+- [Samples](#samples)
+- [Supported Services](#supported-services)
+- [Supported Workloads](#supported-workloads)
 - [Architecture](#architecture)
   - [Controller](#controller)
   - [Webhooks](#webhooks)
-- [Supported Services](#supported-services)
-- [Supported Workloads](#supported-workloads)
-- [Getting Started](#getting-started)
-  - [Running on the cluster](#running-on-the-cluster)
-  - [Undeploy controller](#undeploy-controller)
-- [Samples](#samples)
 - [Contributing](#contributing)
   - [Test It Out](#test-it-out)
   - [Modifying the API definitions](#modifying-the-api-definitions)
 - [Community, discussion, contribution, and support](#community-discussion-contribution-and-support)
   - [Code of conduct](#code-of-conduct)
 
-## Architecture
+## Getting Started
 
-The [Service Binding for Kubernetes Specification](https://servicebinding.io/spec/core/1.0.0/) defines the shape of [Provisioned Services](https://servicebinding.io/spec/core/1.0.0/#provisioned-service), and how the `Secret` is [projected into a workload](https://servicebinding.io/spec/core/1.0.0/#workload-projection). The spec says less (intentionally) about how this happens.
+You’ll need a Kubernetes cluster to run against. You can use [KIND](https://sigs.k8s.io/kind) to get a local cluster for testing, or run against a remote cluster.
 
-Both a controller and mutating admission webhook are used to project a `Secret` defined by the service referenced by the `ServiceBinding` resource into the workloads referenced. The controller is used to process `ServiceBinding`s by resolving services, projecting workloads and updating the status. The webhook is used to prevent removal of the workload projection, projecting workload on create, and a notification trigger for `ServiceBinding`s the controller should process.
+After the controller is deployed, try out the [samples](#samples).
 
-The apis, resolver and projector packages are defined by the reference implementation and reused here with slight modifications. The bulk of the work to bind a service to a workload is encapsulated with these packages. The output from the projector is deterministic and idempotent. The order that service bindings are applied to, or removed from, a workload does not matter. If a workload is bound and then unbound, the only trace will be the `SERVICE_BINDING_ROOT` environment variable.
+### Deploy a released build
 
-There are a limited number of resources that maintain an informer cache within the manager:
-- `ServiceBinding`
-- `ClusterWorkloadResourceMapping`
-- `MutatingWebhookConfiguration`
-- `ValidatingWebhookConfiguration`
+The easiest way to get started is by deploying the [latest release](https://github.com/servicebinding/runtime/releases). Alternatively, you can [build the runtime from source](#build-from-source).
```

Comment:
Now I wish to remove the version number from the release YAML file name. So that we could install the latest version like this:
```
kubectl apply -f https://github.com/servicebinding/runtime/releases/download/latest/servicebinding-runtime.yaml
```
This would be a nice getting started experience.

Reply:
I disagree:

1. the "latest" path segment in the URL is chronological. It's not necessarily the greatest semantic version
2. it's important for users to know what version they are installing as the scope and content of releases will change over time.

Reply:
Good points!